django_fido/management/commands/download_authenticator_metadata.py

```diff
@@ -0,0 +1,54 @@
+"""Command to download metadata for authenticators."""
+from __future__ import unicode_literals
+
+import json
+from base64 import b64decode
+
+import requests
+from django.core.management.base import BaseCommand, CommandError
+from jwcrypto.jwk import JWK
+from jwcrypto.jwt import JWT
+
+from django_fido.models import AuthenticatorMetadata
+from django_fido.settings import SETTINGS
+
+FIDO_MDS_URL = 'https://mds2.fidoalliance.org/'
+
+
+class Command(BaseCommand):
+    """Download metadata for authenticators."""
+
+    help = "Download metadata for authenticators."""
+
+    def add_arguments(self, parser):
+        """Parse command arguments."""
+
+    def handle(self, **options):
+        if SETTINGS.mds_access_token is None:
+            raise CommandError('mds_access_token setting must be specified.')
```

Comment:
It should be `SETTINGS.metadata_service['mds_access_token']` and since it's necessary, shouldn't it be required setting?